<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="graph" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Reprint">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reprint</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBegin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>245</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEnd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>275</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="xBegin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>245</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="xEnd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>275</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
